lib/pack.js

```diff
@@ -163,6 +163,7 @@ function packGitDep (manifest, dir) {
       const child = cp.spawn(process.env.NODE || process.execPath, [
         require.main.filename,
         'install',
+        '--only=dev',
```

Comment:
I did originally have --dev but when I used it locally it told me that it's deprecated and that i should be using --only=dev so I assumed the name was misleading :)